test/test.ts

```diff
@@ -15,7 +15,7 @@ fixture('Pokemon Showdown')
     })
     .beforeEach(t => t.maximizeWindow());
 
-const expectedTeam = `=== Untitled 1 ===
+const expectedTeam = `=== [gen7] Untitled 1 ===
```

Comment:
I'm probably most undecided about this part, the part where the export lists the gen. I think this is probably good.